pymare/estimators/estimators.py

```diff
@@ -594,14 +590,7 @@ def compile(self):
             theta ~ normal(0, tau2);
         }
         """
-        try:
-            from pystan import StanModel
-        except ImportError:
-            raise ImportError(
-                "Please install pystan or, if using Python 3.7+, switch to Python 3.6."
-            )
```

Comment:
pystan and arviz are still optional dependencies:

https://github.com/neurostuff/PyMARE/blob/19af39959baa34eea4a34aa564945a0cbef30f62/setup.cfg#L73-L75

Reply:
I don't know how big of a dependency each one is, but I would lean toward keeping them optional, since they were optional before they started breaking.

Reply:
Oh, good catch. Thanks!